packages/calypso-e2e/src/lib/flows/gutenboarding-flow.ts

```diff
@@ -201,10 +205,7 @@ export class GutenboardingFlow {
 	async selectPlan( name: Plans ): Promise< void > {
 		// First, expand the accordion.
 		await this.page.click( ':text-is("Show all plans")' );
-		await Promise.all( [
-			this.page.waitForNavigation(),
-			this.page.click( selectors.selectPlanButton( name ) ),
```

Comment:
Just a double check that you meant to remove this navigation wait!

Reply:
Yep, with the updated flow, selecting the Plan no longer triggers the creation of a site (since an account needs to be first created).

I've moved up the onus for ensuring navigation to the test writer for now, but have verified that tests do work fine without the `waitForNavigation`.